app/src/app/navtitle/navtitle.component.sass

```diff
@@ -29,7 +29,7 @@ h1
 
 h2
   font: 400 12pt 'Livvic'
-  margin: 0 .5em
+  margin: -0.5em 0em 0em 0.5em
```

Comment:
Things are not aligned anymore. On large screens, h2 is not aligned with h1. On narrow screens, the space below h2 is much shorter than the space above h2, which is itself much shorter than the interline space.

Reply:
Can we reduce the size of h2, "Polls" and "Create"？ So that h2 can be displayed on a ligne in all sizes of screens and aligned downward with h1. 
The code and results are as follows ：
<img width="251" alt="code_navtitle" src="https://user-images.githubusercontent.com/78666554/119118392-6e7cd680-ba2a-11eb-8e0f-ae826a2beb42.png">
<img width="454" alt="navtitle" src="https://user-images.githubusercontent.com/78666554/119119568-991b5f00-ba2b-11eb-8104-f41177ccbd04.png">
<img width="750" alt="navtitle2" src="https://user-images.githubusercontent.com/78666554/119119580-9b7db900-ba2b-11eb-9425-8fb6dddcd2df.png">

Reply:
Changing the size of Polls and Create independently of the width of the screen is not possible, because we want that size to be at least the size of normal text, and we don't want the size of normal text to be less than 13pt on large screen (it has been discussed with Umberto some month ago). Changing the size of all fonts depending on the width of the screen is possible though.